docs/reference/api-conventions.asciidoc

```diff
@@ -675,22 +675,15 @@ should also be passed with a media type value that indicates the format
 of the source, such as `application/json`.
 
 [float]
-=== Content-Type auto-detection
+=== Content-Type Requirements
 
-deprecated[5.3.0, Provide the proper Content-Type header]
+The type of the content sent in a request body must be specified using
+the `Content-Type` header. The value of this header must map to one of
+the supported formats (JSON, YAML, CBOR, SMILE, and NDJSON).
```

Comment:
we do throw error when one sets CBOR or YAML using _bulk, or NDJSON using e.g. search ?